wp-content/themes/csisnuclearnetwork/template-parts/block-post-related.php

```diff
@@ -13,14 +13,21 @@
 
 ?>
 <article <?php post_class('post-block-related'); ?> id="post-<?php the_ID(); ?>">	
-		<?php if ( has_post_thumbnail() ) : ?>
-			<a href="<?php the_permalink(); ?>" class="post-block-related__img" title="<?php the_title_attribute(); ?>">
-				<?php the_post_thumbnail( 'large' ); ?>
-			</a>
-		<?php endif; ?>
-
+	<h4 class='post-block-related__subtitle'><a class="text-strong">ANALYSIS/</a><span><?php nuclearnetwork_display_categories() ?></span></h4>
```

Comment:
Once you do the rebase you'll be able to call the subtype function to do this for you.
```suggestion
<?php nuclearnetwork_display_subtypes(); ?>
```